bin/normalize_anth.py

```diff
@@ -159,7 +159,7 @@ def process(oldnode, informat):
     if not root.tail:
         # lxml drops trailing newline
         root.tail = "\n"
-    for paper in root.findall('paper'):
+    for paper in root.findall('.//paper'):
         fullid = "{}-{}".format(root.attrib['id'], paper.attrib['id'])
```

Comment:
I don't think this will work to get the full id. Instead, get the volume id with

    paper.getparent().attrib['id']

and then pass all three (root id, volume id, and paper id) to `utils/build_anthology_id()` to get the ID.

Reply:
For more info: take an Anth ID "P18-1001" which is `{collection_id}-{volume_id}{zero padded paper_id}`. Volume and Paper IDs are no longer zero-padded in the XML so you have to apply that yourself.